Draft for release blog 2.32 (#5243)
FlowFuse/website: src/blog/2026/07/flowfuse-release-2-32.md

For the full picture of what certification covers and when it is worth it, see [Why FlowFuse Certified Nodes?](/blog/2026/06/flowfuse-certified-nodes/), and browse the current catalog on the [integrations page](https://flowfuse.com/integrations/?certified=1).

Certified Nodes are available to Teams and Enterprise tier customers. New instances get the catalog automatically, and you can contact us to add Certified Nodes to an existing instance.

![A certified node in the palette manager](./images/certified-node-palette.png){data-zoomable}
_TODO asset: a certified node in the palette manager_

### In practice

- You connect to more systems with nodes FlowFuse tests and supports, not community packages you vet yourself
- You reach certified nodes directly in the palette manager, with no separate workflow
- You build on a node knowing it has cleared defined quality and security checks

> ## Pipelines Connect to Any Git Server

Reviewer: Hey @allthedoll @dimitrieh added this section for git integrations - lmk thanks!

Author: @n-lark LGTM but we'll need to add this to the meta data and opening because it's a Pretty Big Deal

Reviewer: Will add and report back!

Reviewer: @allthedoll done lmk!

Author: Perf.

---
title: "FlowFuse 2.32: Certified Redis, Git Pipelines for Any Server, Insights on Remote Instances, and Dark Mode"
subtitle: "Connect DevOps pipelines to any Git server, ask your live edge data questions in plain language, and give your eyes a break on long sessions."
description: "FlowFuse 2.32 certifies the Redis node, extends DevOps Git pipelines to any HTTPS server, and adds Insights mode on remote instances, plus dark mode, Device Agent 4, and a Plan Mode for the Expert."
date: 2026-07-02
authors: ["jamie-strusz"]

> # image: /blog/2026/07/images/flowfuse-release-2-32.png  # uncomment when hero art lands; image-handler stats this path locally so it must be a real file, not a URL (falls back to default blog tile until then)

Reviewer 2: ```suggestion
image: /blog/2026/07/images/flowfuse-release-2-32.jpg
```

<!--more-->

## Certified Redis Integration

Community nodes are code you have to vet, secure, and maintain yourself. Certified Nodes take that on with a defined quality, security, and support process.

### Redis Joins the Catalog

2.32 adds **Redis** as a certified node, so you can integrate with Redis using a node FlowFuse tests, secures, and supports, backed by the same trust contract as every certified node: vetted quality, a CVE response commitment, and a real path to support.

Certified Nodes belong to specific FlowFuse packages. Browse the full catalog on the [integrations page](https://flowfuse.com/integrations/?certified=1), and contact us to get them enabled for your team or instance.

## Pipelines Connect to Any Git Server

DevOps Pipeline Git stages only worked with GitHub or Azure DevOps, leaving self-hosted GitLab or on-prem Bitbucket out.

> Git Repository stages now connect to any HTTPS Git server: GitLab, Bitbucket, Gitea, or a self-hosted instance. Point a pipeline at the repository and it pushes and pulls snapshots as before. For a server behind a private CA, paste in its certificate, no infrastructure changes needed. Your team keeps its existing GitLab or Bitbucket workflow and its compliance setup, with no migration to GitHub.

Reviewer 3: ```suggestion
Git Repository stages now connect to any HTTPS Git server: GitLab, Bitbucket, Gitea, or a self-hosted instance. Point a pipeline at the repository and it pushes and pulls snapshots as before. For a server behind a private CA, paste in its certificate, no infrastructure changes needed. This means your team can integrate seamlessly with your existing Git workflow and its compliance setup.
```
